deepinv/models/artifactremoval.py

```diff
@@ -1,20 +1,22 @@
 import torch
 import torch.nn as nn
+from .base import Reconstructor, Denoiser
 
 
-class ArtifactRemoval(nn.Module):
+class ArtifactRemoval(Reconstructor):
```

Comment:
We should add an option to have identity for artifact removal. And also maybe put less emphasis on FBPConvNet because ArtifactRemoval is very general and essentially just wraps any model with the API `model(y, physics)`

Reply:
done